mullvad-cli/src/format.rs

```diff
@@ -225,6 +225,17 @@ fn print_error_state(error_state: &ErrorState) {
             println!("Blocked: {cause}");
             println!("Your kernel might be terribly out of date or missing nftables");
         }
+        #[cfg(target_os = "macos")]
+        cause @ talpid_types::tunnel::ErrorStateCause::NeedFullDiskPermissions => {
+            println!("Blocked: {cause}");
+            println!();
+            println!(r#"Enable "Full Disk Access" for "Mullvad VPN" in the macOS system settings:"#);
+            println!(r#"open "x-apple.systempreferences:com.apple.preference.security?Privacy_AllFiles"#);
+            println!();
+            println!("Restart the Mullvad daemon for the change to take effect:");
+            println!("launchctl unload -w /Library/LaunchDaemons/net.mullvad.daemon.plist");
+            println!("launchctl load -w /Library/LaunchDaemons/net.mullvad.daemon.plist");
```

Comment:
Why not write it using a single `println!`?
```suggestion
            println!(r#"
            Enable "Full Disk Access" for "Mullvad VPN" in the macOS system settings:
            open "x-apple.systempreferences:com.apple.preference.security?Privacy_AllFiles

            Restart the Mullvad daemon for the change to take effect:
            launchctl unload -w /Library/LaunchDaemons/net.mullvad.daemon.plist
            launchctl load -w /Library/LaunchDaemons/net.mullvad.daemon.plist
            "#);
```